sdk/identity/Azure.Identity/README.md

```diff
@@ -238,6 +238,8 @@ using AzureEventSourceListener listener = AzureEventSourceListener.CreateConsole
 
 All credentials can be configured with diagnostic options, in the same way as other clients in the SDK.
 
+> CAUTION: Requests and responses in the Azure Identity library contain sensitive information. Precaution must be taken to protect logs when customizing the output to avoid compromising account security.
+
 ``` c#
 DefaultAzureCredentialOptions options = new DefaultAzureCredentialOptions()
```

Comment:
```suggestion
var options = new DefaultAzureCredentialOptions
```

Reply:
I thought we generally avoid using `var` in our samples.